.travis.yml

```diff
@@ -50,6 +50,7 @@ install:
 
   - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION -y
   - source activate test-environment
+  - pip install .
   - pip install -r requirements.txt
```

Comment:
@jboes I had to re-ad this to make a test work.

Reply:
Ok, we shouldn't need this and `pip install -r requirements.txt`. The setup file already identifies the requirements from this file and installs them when running `pip install .`

Can you please remove the line below.